cluster/stone-prd-rh01/tenants/orchestrator-releng-tenant/operator/integration_test_scenario.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: appstudio.redhat.com/v1beta1
+kind: IntegrationTestScenario
+metadata:
+  name: orchestrator-helm-operator-enterprise-contract
+spec:
+  params:
+    - name: POLICY_CONFIGURATION
+      value: rhtap-releng-tenant/tmp-onboard-policy
```

Comment:
this exists?

Reply:
I'm not sure, I was following the onboarding documentation as it contains an example with this value. I've also seen it in these [2 other manifests](https://github.com/search?q=repo%3Aredhat-appstudio%2Ftenants-config%20rhtap-releng-tenant%2Ftmp-onboard-policy&type=code) for the insight-runtimes-tenant.

Reply:
I see it!